compio-runtime/src/runtime/mod.rs

```diff
@@ -69,8 +70,8 @@ impl RuntimeInner {
         self.id
     }
 
-    // Safety: the return runnable should be scheduled.
-    unsafe fn spawn_unchecked<F: Future>(&self, future: F) -> Task<F::Output> {
+    // Safety: be careful about the captured lifetime.
```

Comment:
Be more warning and detailed. Something like "Lifetime of future, particularly its captured variable if there's any, must outlive the spawned task" and users should understand that this is very dangerous and hard to guarantee.

Reply:
Here the comment is for the developers. The safety note for users are below, at `Runtime::spawn_unchecked`.

Reply:
I see